integration/src/ethereum/ethereum.ts

```diff
@@ -249,8 +249,12 @@ export function ethereumTests(get: () => { wallet: core.HDWallet; info: core.HDW
         if (!wallet) return;
         // eslint-disable-next-line jest/no-jasmine-globals, jest/no-disabled-tests
 
+        if (ledger.isLedger(wallet)) return; // FIXME: just test kk and native for now
+        if (trezor.isTrezor(wallet)) return; // FIXME: just test kk and native for now
+        if (portis.isPortis(wallet)) return; // FIXME: just test kk and native for now
```

Comment:
Nothing new here, if you open this module, we consistently exclude these including for Ethereum, I reckon the rationale is these aren't properly implemented yet/anymore?

Reply:
Huh, ok, well thanks for the info. Will approve as is then